packages/forma-36-website/gatsby-config.js

```diff
@@ -74,6 +74,10 @@ module.exports = {
                 name: 'Frequently used verbs',
                 link: '/guidelines/copy/frequently-used-verbs/',
               },
+              {
+                name: 'Date and time',
+                link: '/guidelines/copy/date-and-time/',
```

Comment:
Wondering if this deserves its own section rather than being put together with copy.  I thought the links weren't working initially.

Reply:
I'm not sure about that, because it doesn't seem to fit one level up — between Accessibility, Copy and Layout guidelines — and by the nature of the issue, it feels like writing guidelines.

Reply:
I think it fits pretty well. My arguments for the change are:

- The information would be visible sooner (no need to go via the copy path, a plus for discoverability).
- These guidelines can be used in isolation, and are likely to be used this way. If I want to display the created date for an entity in an application, I wouldn't need to know anything about sentence structure or grammar rules.